framework/file_helper.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 class FileHelper
-  def self.delete_tmp(dir)
-    OnlyofficeLoggerHelper.log('Clear tmp dir')
-    FileUtils.rm_rf(Dir.glob(dir.to_s))
+  # @param [String] tmp_source_file is a path to the temporary folder with source file
+  # @param [String] tmp_converted_file is a path to the temporary converted file
+  def self.delete_tmp(tmp_source_file, tmp_converted_file)
+    OnlyofficeLoggerHelper.log('Clear tmp files')
+    FileUtils.rm_rf(Dir.glob(tmp_source_file.to_s))
+    FileUtils.rm_rf(tmp_converted_file.to_s)
```

Comment:
This should be a single file, so safe is not to use `rm_rf` but something like `File.delete`

And I think there is no point to making it `.to_s`